llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Transforms/Transforms.cpp

```diff
@@ -294,10 +294,9 @@ SCFTileAndFusePattern::matchAndRewrite(TilingInterface rootOp,
     }
 
     // Add more fusion candidates to the worklist.
-    if (auto fusedProducerOp =
-            fusedProducer->tiledAndFusedProducer.getDefiningOp()) {
-      addCandidateSlices(fusedProducerOp, candidates);
-      tiledOps.push_back(fusedProducerOp);
+    for (auto tiledOp : fusedProducer->tiledOps) {
+      addCandidateSlices(tiledOp, candidates);
+      tiledOps.push_back(tiledOp);
     }
```

Comment:
I think we will need a similar change in

https://github.com/openxla/iree/blob/d9b052d57713122b8c74a89aa4f383b3063105d4/compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUTileAndFuse.cpp#L189-L195

Reply:
fixed